decoder/gym.go

```diff
@@ -402,6 +403,7 @@ func createGymWebhooks(oldGym *Gym, gym *Gym) {
 			InBattle:       func() bool { return gym.InBattle.ValueOrZero() != 0 }(),
 		}
 
+		external.UpdateGymCount(areas)
```

Comment:
The description says count of newly discovered gyms, but this is not a newly discovered gym - but rather any gym change. Including whether it is in battle or not

Reply:
You are right, I would rework to
```go
if oldGym == nil {
	external.UpdateGymCount(areas)
}
```

Reply:
Yes, so this feels like this would be better inside the saveGym (or updateGym) rather than being part of a webhook function? As you want to do this for all updated gyms regardless of webhooks.
Like pokemon, it could calculate the areas in the save record and pass the areas into the gym webhook function. I think the prometheus update would make sense actually where we are storing the record rather than just dumped randomly in the webhook sending code?

Reply:
However if this is new gyms, whats the difference between this counter and the one you added to fort changes?

Reply:
Hmm, technically nothing. I guess it's safe to remove :) good eye.

Reply:
That's why I wondered if this was a 'gym changes' counter, rather than a new gym counter.

Reply:
No, no... just silly mistake :)